packages/api-nest/src/library/resolvers/rss-feed.resolver.ts

```
@@ -0,0 +1,180 @@
+    } catch (error) {
+      return {
+        success: false,
+        message: 'Failed to update feed settings',
+        errors: [error instanceof Error ? error.message : 'Unknown error'],
+      }
+    }
+  }
+
+  /**
+   * Resolve unread count for a feed
+   */
```

ESLint

[error] Missing JSDoc @returns for function.

(valid-jsdoc)